### PR TITLE
Log carbonapi uuid and request headers

### PR DESCRIPTION
**Pull request by msaf1980**
Some simplify find query info:
1) Log carbonapi_uuid in complete query chain
2) Allow to log additional response headers (for example, may be in future grafana request id for completely trace queries)

### CODE REVIEW

File: graphite-clickhouse.go [hunk 1]
```diff
@@ -59,7 +60,7 @@ func WrapResponseWriter(w http.ResponseWriter) *LogResponseWriter {
 	return &LogResponseWriter{ResponseWriter: w}
 }
 
-func Handler(handler http.Handler) http.Handler {
+func Handler(handler http.Handler, cfg *config.Config) http.Handler {
```
Comment (Felixoid) [inline graphite-clickhouse.go:63]: Isn't it the only place to change? Why all other handlers (capabilities, find, etc..) are changed too? It should duplicate the logging then.

File: graphite-clickhouse.go [hunk 2]
```diff
@@ -59,7 +59,11 @@ func WrapResponseWriter(w http.ResponseWriter) *LogResponseWriter {
 	return &LogResponseWriter{ResponseWriter: w}
 }
 
-func Handler(handler http.Handler) http.Handler {
+type App struct {
+	config *config.Config
+}
+
+func (app *App) Handler(handler http.Handler) http.Handler {
```
Comment (Felixoid) [inline graphite-clickhouse.go:66]: These are unnecessary changes, right?

Comment (msaf1980) [reply graphite-clickhouse.go:66]: With app code is simple - no need to pass config as Handler function parameter. But can be removed.
